README.md

```diff
@@ -3,10 +3,12 @@ This tool is used to streamline the creation of resources for a Synapse study pr
 This tool is intended for Bridge Downstream developers.
 
 ## Getting Started
-Instructions on how to get started with this tool as well as troubleshooting, contributing, etc. are described on the [Bridge analytics template tool documentation page](https://sagebionetworks.jira.com/wiki/spaces/BD/pages/2758410253/Bridge-analytics-template+tool+documentation).
+Instructions on how to get started with this tool as well as troubleshooting, contributing, etc. are described on the [Bridge analytics template tool documentation page](https://sagebionetworks.jira.com/wiki/spaces/BD/pages/2758410253/Bridge-analytics-template+tool+documentation). This page is only accessible to Sage employees.
```

Comment:
The Bridge Downstream confluence space is configured to be publicly accessible. So the last sentence is incorrect.